directives/demo/duplicate-demo.js

```diff
@@ -0,0 +1,14 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { duplicateOffscreen } from '../../../directives/duplicate-offscreen.js';
+
+class DuplicateDemo extends LitElement {
+	render() {
+		const item = html`<div>This message is present in the DOM twice</div>`;
+
+		return html`
+			${duplicateOffscreen(item)}
+		`;
+	}
+}
```

Comment:
Being a duplicate, I would imagine that this would be valid...?
```
render() {
		const item = html`<div>This message is present in the DOM twice</div>`;

		return html`
			item
			${duplicateOffscreen(item)}
		`;
	}
```

Reply:
... or does it insert it into the DOM on-screen, and duplicate it for offscreen?